app/admin/consultation.rb

```diff
@@ -0,0 +1,10 @@
+ActiveAdmin.register_page "Consultation" do
```

Comment:
This file is generated by running the rake task as follows:
`bundle exec rake generate_active_admin_resources[consultation]`